spec/index.html

```diff
@@ -388,11 +388,19 @@ <h2>Definition</h2>
   <h3>Document Element</h3>
   <p>The <a href="#defn-srd">SPARQL Results Document</a> begins with <code>sparql</code> document element in the <code>http://www.w3.org/2005/sparql-results#</code> namespace, written as follows:</p>
   <pre>&lt;?xml version="1.0"?&gt;
-&lt;sparql xmlns="http://www.w3.org/2005/sparql-results#"&gt;
+&lt;sparql xmlns="http://www.w3.org/2005/sparql-results#"
+  xmlns:its="http://www.w3.org/2005/11/its" 
+  its:version="2.0"&gt;
  ...
 &lt;/sparql&gt;
 </pre>
```

Comment:
I would maybe state that other `its:version` values are ok to not make migrating to new `its` version a breaking change in future versions of XML query results.